app/server.go

```diff
@@ -9,9 +9,11 @@ import (
 	"github.com/zillow/howwegoatzillow/libs/server"
 )
 
+type MyServer struct{ *server.Server }
```

Comment:
if we use type alias, then in your Tests you will have to cast everytime since it needs ServeHttp() method which is on server.Server

Reply:
Fwiw, this is not a type alias.

Reply:
> Fwiw, this is not a type alias.

thats what I meant, cannot use a typeAlias since then tests require casting.

Reply:
Do you mean a named type? A type alias `type Foo = Bar` means the types are the exact same, like `uint8` and `byte` being interchangeable.

Reply:
:disappointed: , yes, sorry, I was confused between named type and type alias terminology. 
basically 
(type alias)
```
type Myserver = server.Server
```
does not work. Wire does not distinguish between the two types and complains `multiple bindings for server.Server` found.

(new type)
```
type Myserver server.Server
```
works, but everywhere you need casting when we want to call underlying server.Server methods.